tests/test_dag_growing_network.py

```diff
@@ -58,7 +63,7 @@ def test_init_empty_graph(self) -> None:
         assert self.net.dag.nodes["end"]["size"] == self.out_features
         assert self.net.dag.nodes["start"]["type"] == "L"
         assert self.net.dag.nodes["end"]["type"] == "L"
-        # assert self.net.dag.nodes["end"]["use_batch_norm"] == False
+        assert self.net.dag.nodes["end"]["use_batch_norm"] == False
```

Comment:
```suggestion
        assert self.net.dag.nodes["end"]["use_batch_norm"] is False
```
i think that you are trying to test that the value is *exactly* False. This is slightly different: 
```
0 is False 
>>> False
0 == False 
>>> True
False is False
>>> True
```